### PR TITLE
Add golden metrics for AWS DocDb, Beanstalk and EMR

### PR DESCRIPTION
**Pull request by jlegoff**
### Relevant information

Added golden metric and tags for AWS DocDb, Beanstalk and EMR

### Checklist

* [ ] I've read the guidelines and understand the acceptance criteria.
* [ ] The value of the attribute marked as `identifier` will be unique and valid. 
* [ ] I've confirmed that my entity type wasn't already defined. If it is I'm providing an
 explanation above.

### CODE REVIEW

**Review by cianBuckley — APPROVED** [review]

File: definitions/infra-awselasticbeanstalkenvironment/summary_metrics.yml [hunk 1]
```diff
@@ -1,8 +1,3 @@
-providerAccountName:
```
Comment (naxhh) [inline definitions/infra-awselasticbeanstalkenvironment/summary_metrics.yml:1]: What is the reason to remove this one? I think is provided in pretty much all AWS summaries as you can see [here](https://github.com/newrelic-experimental/entity-synthesis-definitions/search?q=providerAccountName)

File: definitions/infra-awselasticbeanstalkinstance/summary_metrics.yml [hunk 2]
```diff
@@ -1,8 +1,3 @@
-providerAccountName:
```
Comment (naxhh) [inline definitions/infra-awselasticbeanstalkinstance/summary_metrics.yml:1]: same as previous comment

Comment (jlegoff) [reply definitions/infra-awselasticbeanstalkinstance/summary_metrics.yml:1]: it's not a metric, should it still be in there?

Comment (gvassallo) [reply definitions/infra-awselasticbeanstalkinstance/summary_metrics.yml:1]: In the summary metric section, you can also specify tags which will be displayed in the explorer as one of the columns. 
https://github.com/newrelic-experimental/entity-synthesis-definitions/blob/main/docs/summary_metrics.md#tag

Comment (jlegoff) [reply definitions/infra-awselasticbeanstalkinstance/summary_metrics.yml:1]: got it, I'll put them back!

**Review by naxhh — APPROVED** [review]

**Pull request merged**